### ISSUE TITLE
Nit in invariants draft

### ISSUE THREAD

**Issue by martinduke** [quicwg/base-drafts#3635]
Both section 4.2 and 4.3 say

> Bits that have version-specific semantics are marked with an X.

Unfortunately, the [new HTML format](https://quicwg.org/base-drafts/draft-ietf-quic-invariants.html#fig-long) doesn't have Xs. I'm not sure how to reword this to support both formats.

**Comment by LPardue** [comment]
how about `Bits that have version-specific semantics are called Version-Specific Bits` :)

**Comment by LPardue** [comment]
I think this was fixed by https://github.com/quicwg/base-drafts/pull/3636, I dont know if that was just a coincidence or it was untagged

**Comment by martinthomson** [comment]
That was intentional.  I just neglected to tag this.
